rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md

```diff
@@ -0,0 +1,41 @@
+# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
+
+Git hash: `3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d`
+
+New compressed Wasm hash: `ccda8966249bc7d89f14968a242d6fb2c33d2363336303ec4f12977cab74c6f7`
+
+Target canister: `vxkom-oyaaa-aaaar-qafda-cai`
+
+Previous ledger suite orchestrator proposal: https://dashboard.internetcomputer.org/proposal/131755
+
+---
+
+## Motivation
+This proposal upgrades the ckERC20 ledger suite orchestrator to add support for [XAUt](https://etherscan.io/token/0x68749665FF8D2d112Fa859AA293F07A622782F38#tokenInfo). Once executed, the twin token ckXAUT will be available on ICP, refer to the [documentation](https://github.com/dfinity/ic/blob/master/rs/ethereum/cketh/docs/ckerc20.adoc) on how to proceed with deposits and withdrawals.
```

Comment:
@manudrijvers The ERC-20 token is name `XAUt` (note the lowercase t), but I guess we want to name the ck token, `ckXAUT` (upper-case T), correct?